src/kv/store.h

```diff
@@ -325,14 +325,16 @@ namespace kv
         auto search = maps.find(map_name);
         if (search == maps.end())
         {
-          LOG_FAIL_FMT("No such map {} at version {}", map_name, v);
+          LOG_FAIL_FMT("failed to deserialize");
```

Comment:
I don't mind either way, but I think we should either capitalise or not capitalise error messages.